src/dataplane-protocol/src/record.rs

```diff
@@ -341,42 +341,6 @@ where
     }
 }
 
-impl<B> From<String> for Record<B>
```

Comment:
Why remove this?

Reply:
Previously, we were using `Record` almost as if it were a newtype wrapped around `value`, a single piece of data in the Record. Now that we are using both key and value, it does not make sense to me that we have all of these conversions that are heavily biased towards `value` and ignore `key`. Plus this was largely unused anyways.

Reply:
It still useful and this breaks contract with previous version unnecessary.

Reply:
I went ahead and put this back 👍